src/main/java/beans/QuestionBean.java

```diff
@@ -108,11 +113,14 @@ public void setDatatype(String datatype) {
         this.datatype = datatype;
     }
 
-    public StyleBean getStyle() {
-        return style;
+    public HashMap<String, String> getStyle() {
+        if(style != null) {
+            return style;
+        }
+        return new HashMap<String, String>();
```

Comment:
should `style` just be initialized as `new HashMap<String, String>()` so we can just `return style`